ramls/request.json

```diff
@@ -172,6 +172,28 @@
           "description": "Elements of a full call number",
           "$ref": "schema/call-number-components.json",
           "readonly": true
+        },
+        "identifiers": {
+          "type": "array",
+          "description": "An extensible set of name-value pairs of identifiers associated with the resource",
```

Comment:
Why is it relevant that these are name value pairs? Could this be described in domain language rather than technical language instead?

Reply:
That is the same description as instance have.

Does this description work better:
`An extensible set of identifier types and values associated with the resource`?

Reply:
Your suggestion would be fine, my suggestion would be something like `set of resource identifiers`